src/platform/PICO/io_def_generated.h

```diff
@@ -0,0 +1,230 @@
+#define DEFIO_TAG__PA2    DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 2)
+#define DEFIO_TAG__PA3    DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 3)
+#define DEFIO_TAG__PA4    DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 4)
+#define DEFIO_TAG__PA5    DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 5)
+#define DEFIO_TAG__PA6    DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 6)
+#define DEFIO_TAG__PA7    DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 7)
+#define DEFIO_TAG__PA8    DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 8)
+#define DEFIO_TAG__PA9    DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 9)
+#define DEFIO_TAG__PA10   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 10)
+#define DEFIO_TAG__PA11   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 11)
+#define DEFIO_TAG__PA12   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 12)
+#define DEFIO_TAG__PA13   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 13)
+#define DEFIO_TAG__PA14   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 14)
+#define DEFIO_TAG__PA15   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 15)
+#define DEFIO_TAG__PA16   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 16)
+#define DEFIO_TAG__PA17   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 17)
+#define DEFIO_TAG__PA18   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 18)
+#define DEFIO_TAG__PA19   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 19)
+#define DEFIO_TAG__PA20   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 20)
+#define DEFIO_TAG__PA21   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 21)
+#define DEFIO_TAG__PA22   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 22)
+#define DEFIO_TAG__PA23   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 23)
+#define DEFIO_TAG__PA24   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 24)
+#define DEFIO_TAG__PA25   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 25)
+#define DEFIO_TAG__PA26   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 26)
+#define DEFIO_TAG__PA27   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 27)
+#define DEFIO_TAG__PA28   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 28)
+#define DEFIO_TAG__PA29   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 29)
+#define DEFIO_TAG__PA30   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 30)
+
+#if defined(RP2350B)
+#define DEFIO_TAG__PA31   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 31)
+#define DEFIO_TAG__PA32   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 32)
+#define DEFIO_TAG__PA33   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 33)
+#define DEFIO_TAG__PA34   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 34)
+#define DEFIO_TAG__PA35   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 35)
+#define DEFIO_TAG__PA36   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 36)
+#define DEFIO_TAG__PA37   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 37)
+#define DEFIO_TAG__PA38   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 38)
+#define DEFIO_TAG__PA39   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 39)
+#define DEFIO_TAG__PA40   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 40)
+#define DEFIO_TAG__PA41   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 41)
+#define DEFIO_TAG__PA42   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 42)
+#define DEFIO_TAG__PA43   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 43)
+#define DEFIO_TAG__PA44   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 44)
+#define DEFIO_TAG__PA45   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 45)
+#define DEFIO_TAG__PA46   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 46)
+#define DEFIO_TAG__PA47   DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 47)
+#endif
+
+#define DEFIO_TAG_E__PA0  DEFIO_TAG__PA0
```

Comment:
(This made sense when target defined available pins and configuration was compile-time checked. Maybe it is time to remove it)

Reply:
Possibly... I'm just playing here to promote discussion... given the variance between PICO and STM